source/_includes/hardware-platforms.html

```diff
@@ -1,112 +1,164 @@
-      <td colspan="5">4</td>
-    </tr>
-    <tr>
-      <td>Charging port</td>
-      <td>Power only</td>
-      <td colspan="2">Smart accessory port</td>
-      <td>Smart accessory port <em>(data only)</em></td>
-      <td>Smart accessory port</td>
-    </tr>
-  </tbody>
+    <thead>
+        <tr>
+            <td>Manufacturer</td>
+            <th colspan="5">Pebble</th>
+            <th colspan="2">Core Devices</th>
+        </tr>
+        <tr>
+            <td>Model</td>
+            <th>Classic,<br />Steel</th>
+            <th nowrap="nowrap">Time,<br />Time Steel</th>
+            <th>Time Round</th>
+            <th>Pebble 2</th>
+            <th nowrap="nowrap">Time 2<br /><em>(Not released)</em></th>
+            <th nowrap="nowrap">Pebble 2 Duo<br /><em>(Core 2 Duo)</em></th>
+            <th nowrap="nowrap">Pebble Time 2*<br /><em>(Core Time 2)</em></th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Platform</td>
+            <td>Aplite</td>
+            <td>Basalt</td>
+            <td>Chalk</td>
+            <td>Diorite</td>
+            <td>Emery</td>
+            <td>Diorite</td>
+            <td>Emery</td>
+        </tr>
+        <tr>
+            <td>CPU</td>
+            <td nowrap="nowrap">Cortex-M3<br />64 MHz</td>
+            <td colspan="3">Cortex-M4<br />100 MHz</td>
+            <td>Cortex-M7<br />144 MHz</td>
+            <td></td>
+            <td>SiFli SF32LB52J</td>
+        </tr>
+        <tr>
+            <td>Max. resource size</td>
+            <td>96k</td>
+            <td colspan="3">256k</td>
+            <td>512k</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Max. app size (code + heap)</td>
+            <td>24k</td>
+            <td colspan="3">64k</td>
+            <td>128k</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Display shape</td>
+            <td colspan="2">Rectangle</td>
+            <td>Round</td>
+            <td colspan="4">Rectangle</td>
+        </tr>
+        <tr>
+            <td>Display size</td>
+            <td nowrap="nowrap">1.26"</td>
+            <td nowrap="nowrap">1.25"</td>
+            <td nowrap="nowrap">1"</td>
+            <td nowrap="nowrap">1.26"</td>
+            <td nowrap="nowrap">1.5"</td>
+            <td nowrap="nowrap">1.26"</td>
+            <td nowrap="nowrap">1.5"</td>
+        </tr>
+        <tr>
+            <td>Display resolution</td>
+            <td colspan="2">144 x 168</td>
+            <td nowrap="nowrap">180 x 180</td>
+            <td nowrap="nowrap">144 x 168</td>
+            <td nowrap="nowrap">200 x 228</td>
+            <td nowrap="nowrap">144 x 168</td>
+            <td nowrap="nowrap">200 x 228</td>
+        </tr>
+        <tr>
+            <td>Display PPI</td>
+            <td>175</td>
+            <td colspan="2">182</td>
+            <td>175</td>
+            <td>202</td>
+            <td>176</td>
```

Comment:
I think this should be 175? Pretty sure P2 and C2D have the same display

Reply:
On the Repebble website, the hardware specs says 176: https://store.repebble.com/

I agree, this could be a mistake. Can anyone confirm?

Reply:
I have set the dimensions for both to 175. I have assumed that is correct.

Reply:
I've [asked for some clarification](https://discord.com/channels/221364737269694464/257631082218782721/1403641569864585236) to make sure we get this one right - will update when I hear back

Reply:
I saw Eric responded saying it is 175.